plugin/web/extensions/sidebar/popups/buildingSearch/components/hooks.ts

```diff
@@ -409,7 +409,7 @@ export default () => {
   useEffect(() => {
     document.documentElement.style.setProperty("--theme-color", "#00BEBE");
     onInit((window as any).buildingSearchInit);
-
+    postMsg({ action: "buildingSearchReady" });
```

Comment:
Can you most this in to the onInit. I feel like it makes sense to be included there.